From 8135bdd48b7eef6b61fb2d47ff7afb13dd06d8f3 Mon Sep 17 00:00:00 2001
Message-Id: <8135bdd48b7eef6b61fb2d47ff7afb13dd06d8f3.1684964119.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1684964119.git.sharlatanus@gmail.com>
References: <cover.1684964119.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 May 2023 23:30:37 +0100
Subject: [PATCH v2 18/20] gnu: python-asdf: Update to 2.15.0.

* gnu/packages/astronomy.scm (python-asdf): Update to 2.15.0.
[native-inputs]: Add python-fsspec.
---
 gnu/packages/astronomy.scm | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index b9fbb9d829..8c5d0455bc 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2729,13 +2729,13 @@ (define-public python-suntime
 (define-public python-asdf
   (package
     (name "python-asdf")
-    (version "2.13.0")
+    (version "2.15.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "asdf" version))
        (sha256
-        (base32 "1zixzv4n2fryaszsfchqh2nvp0gzvarhz03fc721yw6iafdadqij"))))
+        (base32 "11s56797l5330kkhppkyz0bsvms016knmyswj4gx91zrxf8iqvv8"))))
     (build-system pyproject-build-system)
     (arguments
      ;; FIXME: Tests fail a lot with
@@ -2746,6 +2746,7 @@ (define-public python-asdf
      `(#:tests? #f))
     (native-inputs
      (list python-astropy
+           python-fsspec
            python-packaging
            python-psutil
            python-pytest
-- 
2.39.2

